package.json

```diff
@@ -2,41 +2,42 @@
   "name": "@mentormate/node-cli",
   "version": "1.0.2",
   "description": "Node CLI",
-  "bin": {
-    "node-cli": "bin/node-cli"
-  },
   "repository": {
     "url": "https://github.com/MentorMate/node-project-cli"
   },
-  "scripts": {
-    "format": "eslint \"**/*.{js,jsx}\" --fix --cache && prettier \"**/*.{js,jsx,json}\" --write --cache --cache-strategy metadata --cache-location .prettiercache",
-    "test": "jest",
-    "watch": "jest --watch",
-    "snapupdate": "jest --updateSnapshot",
-    "coverage": "jest --coverage"
+  "license": "MIT",
+  "bin": {
+    "node-cli": "bin/node-cli"
   },
   "files": [
     "assets",
     "src",
     "docs",
     "bin"
   ],
-  "license": "MIT",
+  "scripts": {
+    "coverage": "jest --coverage",
+    "eslint": "eslint \"**/*.js\" --fix --cache",
+    "format": "npm run eslint && npm run prettier",
+    "prepare": "husky install",
+    "prettier": "prettier \"**/*.js\" --write --cache --cache-strategy metadata --cache-location .prettiercache",
+    "snapupdate": "jest --updateSnapshot",
```

Comment:
Lets not encourage snapshots, actually we should do the opposite - developers to NOT use snapshots